### DISCUSSION TITLE
2021's Biggest Advancements in Artificial Intelligence

### DISCUSSION THREAD

**Discussion by wh20502** [1101-GS3073-C/Python-Community-GS3073-C#136] [科技新知]
今年又要結束了，這個影片整理了這一年的AI發展，與以前不同的是現在的硬體設備可以盡力跟上軟體更新速度，所以進展的速度比往年更快。影片中介紹五大AI運用進步，包含運用AI(深度學習、機器學習)用以晶片製造，以AI技術解決各行問題等等。看起來滿老生常談的，不過以後進業界應該會比較清楚。
https://www.youtube.com/watch?v=t4B99T_3IsM

**Comment by pocper1** [comment]
也分享一則新聞
- [產業與AI人工智慧](https://findit.org.tw/researchPageV2.aspx?pageId=1798)

**Comment by Timmy110403540** [comment]
希望2022 AI也能繼續在業界上有所創新!
